scripts/linters/pylint_extensions.py

```diff
@@ -488,6 +560,43 @@ class DocstringParameterChecker(checkers.BaseChecker):
     not_needed_param_in_docstring = {'self', 'cls'}
     docstring_sections = {'Raises:', 'Returns:', 'Yields:'}
 
+    # Docstring section headers split up into arguments, returns, yields and
+    # and raises sections signifying that we are currently parsing the
+    # corresponding section of that docstring.
+    DOCSTRING_SECTION_RETURNS = 'returns'
+    DOCSTRING_SECTION_YIELDS = 'yields'
+    DOCSTRING_SECTION_RAISES = 'raises'
+
+    def visit_classdef(self, node):
```

Comment:
Why we have not written in a way similar to other checks and called it from visit_functiondef?

Reply:
Because this check is only for class docstring

Reply:
Could you please add what check it is performing in the docstring as well?

Reply:
Done.